<?xml version="1.0"?>
<launch>
  <arg name="mav_name"      default="firefly"/>
  <arg name="frame_id"      default="world" />
  <arg name="robot_radius"  default="0.45"/>

  <group ns="$(arg mav_name)">
    <node name="frontier_evaluator" pkg="frontier_explorer" type="frontier_evaluator_node" output="screen" clear_params="true" respawn="true">
        <param name="visualize"          value="true" />
        <param name="frame_id"           value="$(arg frame_id)" />
        <param name="accurate_frontiers" value="true" />
        <param name="checking_distance"  value="1.0"/> 
        <param name="robot_radius"       value="0.4" />
        <param name="verbose"            value="false" />
       
        <remap from="frontier_evaluator/esdf_map_in" to="esdf_map" />
        <remap from="frontier_evaluator/tsdf_map_in" to="tsdf_map" />
      </node>
  </group>

</launch>
